mainnet/README.md

```diff
@@ -39,6 +39,10 @@ source .env
 ls $USER_HOME $EIGENLAYER_HOME $EIGENDA_HOME
 ```
 
+:::warning
```

Comment:
same - please update with correct readme

Reply:
done. Same update for both files.